src/pf-notification-drawer/pf-notification-drawer.scss

```diff
@@ -0,0 +1,46 @@
+.drawer-pf {
```

Comment:
For OpenShift IO, we converted all their SCCS to LESS. Mainly so they could build their own CSS along with Patternfly CSS and patternfly-ng. Having patternfly-webcomponents be SCCS while Patternfly is LESS may end up being a show stopper for some? That is, considering they will both be compiled together in apps. At least, OSIO could not consume this as is.